lux-cli/src/debug.rs

```diff
@@ -2,7 +2,20 @@ use crate::{
     project::DebugProject,
     unpack::{Unpack, UnpackRemote},
 };
-use clap::Subcommand;
+use clap::{Args, Subcommand};
+
+#[derive(Args)]
+pub struct Dependencies {
+    /// Output format: human-readable or json
+    #[arg(long, default_value = "human")]
+    pub format: OutputFormat,
+}
+
+#[derive(Clone, clap::ValueEnum)]
+pub enum OutputFormat {
+    Human,
+    Json,
+}
```

Comment:
todo: We already have a `OutputFormat` enum defined in `lux-cli/src/check.rs`.
Let's move that to a `lux-cli/src/args.rs` and reuse it.